.github/workflows/vib-verify.yaml

```diff
@@ -0,0 +1,88 @@
+    outputs:
+      container: ${{ steps.get-container.outputs.container }}
+      result: ${{ steps.get-container.outputs.result }}
+      flavors: ${{ steps.get-container.outputs.flavors }}
+    steps:
+      - id: get-container
+        name: Get modified containers
+        run: |
+          # Using the Github API to detect the files changed as git merge-base stops working when the branch is behind
+          # and jitterbit/get-changed-files does not support pull_request_target
+          URL="https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/files"
+          files_changed_data=$(curl -s --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' -X GET -G "$URL")
+          files_changed="$(echo $files_changed_data | jq -r '.[] | .filename')"
+          # Adding || true to avoid "Process exited with code 1" errors
+          containers_dirs_changed="$(echo "$files_changed" | xargs dirname | grep -o "containers/[^/]*" | sort | uniq || true)"
+          flavors=($(echo "$files_changed" | xargs dirname | grep -o "containers/.*/.*/[^/]*" | sort | uniq || true))
+          flavors_json=$(printf "%s\n" "${flavors[@]}" | jq -R . | jq -cs .)
+          # Using grep -c as a better alternative to wc -l when dealing with empty strings."
+          num_containers_changed="$(echo "$containers_dirs_changed" | grep -c "containers" || true)"
+
+          if [[ "$num_containers_changed" -eq "1" ]]; then
+            # Changes done in only one container -> OK
+            container_name=$(echo "$containers_dirs_changed" | sed "s|containers/||g")
+            echo "::set-output name=result::ok"
+            echo "::set-output name=container::${container_name}"
+            echo "::set-output name=flavors::${flavors_json}"
+          elif [[ "$num_containers_changed" -le "0" ]]; then
+            # Changes done in the containers/ folder but not inside a container subfolder -> SKIP
+            echo "::set-output name=error::No changes detected in containers. The rest of the tests will be skipped."
+            echo "::set-output name=result::skip"
+          else
+            # Changes done in more than container -> SKIP
+            echo -e "::set-output name=error::Changes detected in more than one container directory:\n${containers_dirs_changed}\nIt is strongly advised to change only one container in a PR. The rest of the tests will be skipped."
+            echo "::set-output name=result::skip"
+          fi
+      # Using actions/github-scripts because using exit 1 in the script above would not provide any output
+      # Source: https://github.community/t/no-output-on-process-completed-with-exit-code-1/123821/3
+      - id: show-error
+        name: Show error
+        if: ${{ steps.get-container.outputs.result == 'fail' }}
+        uses: actions/github-script@v3
+        with:
+          script: |
+              core.setFailed('${{ steps.get-container.outputs.error }}')
```

Comment:
This will never be executed, as none of the cases above set `result::fail`. You may safely remove this block of code.